README.md

```diff
@@ -3,24 +3,25 @@
 [![Build Status](https://travis-ci.org/maennchen/ZipStream-PHP.svg?branch=master)](https://travis-ci.org/maennchen/ZipStream-PHP)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/maennchen/ZipStream-PHP/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/maennchen/ZipStream-PHP/)
 [![Code Coverage](https://scrutinizer-ci.com/g/maennchen/ZipStream-PHP/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/maennchen/ZipStream-PHP/)
+[![License](https://img.shields.io/github/license/maennchen/zipstream-php.svg)](LICENSE)
 
-Please see the [LICENSE](LICENSE) file for licensing and warranty information.  The
-latest version of this software is available at the following URL: https://github.com/maennchen/ZipStream-PHP
+## Overview
+
+A fast and simple streaming zip file downloader for PHP. Using this library will save you from having to write the Zip to disk. You can directly send it to the user, which is much faster. It can work with S3 buckets or any PSR7 Stream.
+
+Please see the [LICENSE](LICENSE) file for licensing and warranty information.
+
+The latest version of this software is available at the following URL: https://github.com/maennchen/ZipStream-PHP.
```

Comment:
I would point users to composer instead of downloading it manually.

Reply:
Well, that sentence was there before and I felt weird to me but I left it there. I think it should be removed as to not confuse users into thinking it's a way to install the lib.